includes/sdk-debuggability/browser.md

```diff
@@ -0,0 +1,68 @@
+
+* If you're able to send the event successfully after entering `amplitude.init(API_KEY, 'USER_ID')` in the browser console, it indicates that your `amplitude.init` call might not have been triggered. We support deferred initialization, so the events will be dispatched after the initialization call. Therefore, please check your implementation.
+
+##### Network Request
+
+Use the **Inspect > Network** tab to view all network requests made by your page. Search for the Amplitude request.
+![sdk debuggability network request](../../../assets/images/sdk-debuggability-network-request.png)
+
+Please check the response code and ensure that the response payload is as expected.
+
+##### Instrumentation Explorer/Chrome Extension
+
+The Amplitude Instrumentation Explorer is an extension available in the Google Chrome Web Store. The extension captures each Amplitude event you trigger and displays it in the extension popup. It's important to ensure that the event has been sent out successfully and to check the context in the event payload.
+
+Check [here](../../debugger/#step-2-analyze-the-event-stream) for more details.
+
+#### Common Issues
+
+The following are common issues specific to Browser SDK. For additional general common issues, please refer to [this document](../../sdk-debuggability/).
+
+##### AD Blocker
+
+`Ad Blocker` might lead to event dropping. The following errors indicate that the tracking has been affected by `Ad Blocker`. When loading via a script tag, an error may appear in the console/network tab while loading the SDK script. When loaded with npm package, there could be errors in the network tab when trying to send events to the server. The errors might vary depending on the browser.
+
+* Chrome (Ubuntu, MacOS)
+Console: error net::ERR_BLOCKED_BY_CLIENT
+Network: status (blocked:other)
+* Firefox (Ubuntu)
+Console: error text doesn’t contain any blocking-specific info
+Network: Transferred column contains name of plugin Blocked by uBlock Origin
+* Safari (MacOS)
+Console: error contains text Conent blocker prevented frame ... from loading a resource from ...
+Network: it looks like blocked requests are not listed. Not sure if it’s possible to show them.
+
+We are recomend to using Proxy server to avoid this situation.
+
+##### Cookies related
+
+Here is the [information](./#cookie-management) SDK store in the cookies. This means that client behavior, like disabling cookies or using a private browser/window/tab, will affect the persistence of these saved values in the cookies. So, if these values are not persistent or are not increasing by one, that could possibly be the reason.
+
+##### CORS
+
+If the issue is like the following, it means the issue is related to Cross-Origin Resource Sharing (CORS), a security measure implemented by browsers to restrict how resources on a web page can be requested from a different domain. When you used `setServerURL`, it might have this issue.
```

Comment:
nits:
```suggestion
Cross-Origin Resource Sharing (CORS) is a security measure implemented by browsers to restrict how resources on a web page can be requested from a different domain. When you used `setServerURL`, it might have this issue.
```